python/sensitive_detectors.py

```diff
@@ -75,11 +75,11 @@ class HcalSD(simcfg.SensitiveDetector) :
         considered sensitive. Any volume name containing at least one of these
         identifiers will have a sensitive detector attached.
 
-        The current defaults match the mainline LDMX Hcal (ScintBox) and
+        The current defaults match the mainline LDMX Hcal (ScintBox,ScintVBox,ScintHBox) and
```

Comment:
Yeah we need to make sure that this part matches all of the volumes we are making. The matching looking for raw substrings in the volume names so we will have to list them all in some manner

Reply:
Just a thought, as an "outsider" who hasn't worked much with HCal volumes, I find H and V to be more intuitive. (This might, though, be because when I implemented vertical bars in the TS I was calling them something with X, just because this is the direction of added granularity they provide.) Regardless I agree with @cmantill that it's good to have a clear convention.